.github/workflows/build-package-docs.yaml

```diff
@@ -108,10 +108,16 @@ jobs:
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
 
-    - name: Log the workflow inputs if deploy checked
-      if: fromJSON(github.event.inputs.deploy)
+  check-deploy:
+    if: github.event_name == 'workflow_dispatch' && github.event.inputs.deploy == 'true'
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    outputs:
+      deploy_build: true
```

Comment:
Can you get rid of this too?

```suggestion
```

Reply:
Thanks, yeah. I ran out of time last night and didn't remove that. Just pushed another commit to get rid of it.